### ISSUE TITLE
Problem of notification with the Android last security version of september

### ISSUE THREAD

**Issue by rcrocus** [domoticz/domoticz-android#670]
Hello,

Since I've updated my 2 Android phones with the last security version (Android 13, security version of 05 septemeber 22), no notification are displayed anymore on the phone. When I look in the app the notifications are there but they don't pop up on the phone as they did before.

When I go into the app setting the "Turn on/off notification" is greyed out and can't be toggled and the text "at your request android is blocking this category of notifications".

Before this update, I use Android 13 and the notification display well.

**Comment by mlaurent777** [comment]
Hello,

Same here, I have a Pixel 6 with the latest Android 13 update (at the date of 17/10/2022).
I no longer have Domiticz notifications and when I go in the notifications menu the app I can't enable it.
When I try to turn on the notifications, it's blocked by the system and I have the same message than @rcrocus 

I use the premium version to receive messages from my alarm system based on Domoticz so it's really problematic.

![Android Domoticz notifications menu](https://user-images.githubusercontent.com/108059162/196178187-87ab1870-0b2e-4826-bd0f-993c580b6acd.png)

**Comment by rcrocus** [comment]
Any update or news about this problem ??

**Comment by mlaurent777** [comment]
I have read articles about similar issues on other apps.
The problem would come from the fact that Android 13 has a new system for managing permissions and notifications.

The developers of Domoticz must take this into account and release an update.
Quickly I hope :/

**Comment by Olifant1990** [comment]
Also having this problem with android 13 with november patch. Cannot enable notification.
In the Domoticz app I van send device ID to Domoticz server.
It is then updating the device ID, but when testen, I do not receive a notification. API version is 33 (Android33)

**Comment by galadril** [comment]
Thanks, ill  look into this. 
Related post on permissions:
https://developer.android.com/develop/ui/views/notifications/notification-permission

**Comment by galadril** [comment]
New beta is being validated by Google (should be in the store in like 2 days, please check the beta version)

**Comment by galadril** [comment]
beta subscribe link:
https://play.google.com/apps/testing/nl.hnogames.domoticz

**Comment by cyberictagop** [comment]
Hi all 
Not sure if its oke to mention/post here. 
Since the update to android 13 i also having some issue to control my blinds fron the domoticz app. In stead of open or close it only says stop blind in the app no matter if I press open or close or stop.

**Comment by Olifant1990** [comment]
@cyberictagop Please make a new issue, not related to notification problems ;)

@galadril This morning I received an app update. After the update, I stille didn't see a notifcation when testing.
Then i went to notificiation settings as mentioned in https://github.com/domoticz/domoticz-android/issues/670#issuecomment-1280796697 and I saw it was still disabled. But, now I was able to turn it on.
Now I am receiving notifications again. Thanks!

**Comment by galadril** [comment]
I've moved that topic to a seperate issue

**Comment by galadril** [comment]
Good to know that its fixed then, ill move that release to PROD so others can use it
